crates/mpc-attestation/measurements_build.md

```diff
@@ -0,0 +1,112 @@
+# 📘 TCB Measurements Build Guide
+
+## 📁 Location of JSON Files
+Human-readable TCB measurement JSON files live in:
+
+```
+crates/mpc-attestation/assets/
+```
+
+Typical files:
+
+```
+tcb_info.json
+tcb_info_dev.json
+```
+
+You can add more files in the future (e.g., staging, additional images).  
+Any file matching the prefix:
+
+```
+tcb_info*
+```
+
+will be automatically included at build time.
+
+---
+
+## 📄 JSON File Format
+JSON format is taken directly from the MPC node's `/public_data` endpoint.
+
+Only the following fields under:
+
+```
+tee_participant_info.Dstack.tcb_info
+```
+
+are used:
+
+- `mrtd`
+- `rtmr0`
+- `rtmr1`
+- `rtmr2`
+- key-provider event digest (from the event log)
+
+All other fields are ignored.
+
+---
+
+## ➕ Adding a New Measurements File
+
+1. Obtain the new TCB info JSON from a node and extract the relevant section:
+
+```bash
+curl "http://<node-ip>:<port>/public_data" \
+  | jq -r '.tee_participant_info.Dstack.tcb_info' \
+  > crates/mpc-attestation/assets/tcb_info_new.json
+```
+
+2. Rebuild:
+
+```bash
+cargo clean -p mpc-attestation
+cargo build -p mpc-attestation
+```
+
+The build script will:
+
+- parse the JSON files  
+- extract the required measurement fields  
+- decode them into fixed byte arrays  
+- generate Rust constants  
+- embed them statically into the WASM contract  
```

Comment:
this will not happen as we are not building the contract here, so better remove this line and modify the previous one to say that the constants will be embedded when the contract is built (or something similar)